### PR TITLE
Parameterize 'Ubuntu' in translation strings

### PR DESCRIPTION
**Pull request by jpnurmi**
The parameter values are still hard-coded as 'Ubuntu' until the flavor
name available (#478).

The parameters are all capitalized to match Ubiquity's translation
strings. This way, Weblate is able to automatically suggest matching
translations.

Fixes: #452

### CODE REVIEW

**Comment by jpnurmi** [comment]
I split the change into two commits. The first commit is the actual change and worth reviewing. The second one is the result of `flutter gen-l10n`.

File: packages/ubuntu_desktop_installer/lib/l10n/app_en.arb [hunk 1]
```diff
@@ -1,34 +1,59 @@
...
+    }
+  },
   "releaseNotesLabel": "You may wish to read the <a href=\"{url}\">release notes</a>.",
   "@releaseNotesLabel": {
     "type": "text",
     "placeholders": {
       "url": {}
     }
   },
-
```
Comment (jpnurmi) [inline packages/ubuntu_desktop_installer/lib/l10n/app_en.arb:31]: the Flutter Intl extension for VS Code wants to remove these line-breaks, and Weblate doesn't retain them either, so...

**Comment by jpnurmi** [comment]
I tested with a local Weblate instance that parameter matching is case-sensitive.
| `{RELEASE}` (ARB) vs. `${RELEASE}` (PO) | `{release}` (ARB) vs. `${RELEASE}` (PO) |
|---|---|
| image | image |

**Comment by oSoMoN** [comment]
Why are some of the parameters named `RELEASE` while others are named `DISTRO`? Are we expecting that some will include a version string or a codename?
As far as I can tell, for the Ubuntu desktop installer, their value is always "Ubuntu".

**Comment by jpnurmi** [comment]
I was wondering the same. :) The translation strings are matched with Ubiquity: https://git.launchpad.net/ubiquity/tree/debian/real-po/fr.po

**Comment by oSoMoN** [comment]
Got it, that's coming from Ubiquity. This sounds like historical inconsistency, rather than an intentional distinction.
I suppose fixing this inconsistency now would make the translations import more difficult, but we can make a note to fix it later?

**Comment by jpnurmi** [comment]
Good idea, I created https://github.com/canonical/ubuntu-desktop-installer/issues/486

**Comment by oSoMoN** [comment]
LGTM, thanks!

**Pull request merged**